tools/tests/src/expected/DocExtractionRes.res.json

```diff
@@ -270,5 +270,41 @@
         "col": 3
       }
     }]
+  }, 
+  {
+    "id": "DocExtractionRes.Example",
+    "name": "Example",
+    "kind": "moduleType",
+    "docstrings": [],
```

Comment:
docstrings shouldn't be empty, since the module type contains:
```rescript
module type Example = {
  /***
  this is an example module type 
  */
...
}
```

Reply:
I believe `/***` is for the top level only. If you want a comment on an in-file module you need to use `/**` just above the module definition, like regular docstrings.

Reply:
You confused me for a minute there.. ;) 
I don't think so.
See my simple example:
https://github.com/woeps/rescript-tools-doc-md/blob/54b93c3da527074be767227c057e5b10dbdf7b85/test/ExampleModule.res#L6

https://github.com/woeps/rescript-tools-doc-md/blob/54b93c3da527074be767227c057e5b10dbdf7b85/test/md/Example.md?plain=1#L166